kde-misc/kdialog/kdialog-22.12.2.recipe

```diff
@@ -0,0 +1,118 @@
+SUMMARY="Display dialog boxes from shell scripts"
+DESCRIPTION="kdialog allows you to display dialog boxes from shell scripts.
+The syntax is very much inspired from the 'dialog' command
+(which shows text mode dialogs)."
+HOMEPAGE="https://invent.kde.org/utilities/kdialog/"
+COPYRIGHT="2006-2023 KDialog Team"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://invent.kde.org/utilities/kdialog/-/archive/v22.12.2/kdialog-v22.12.2.tar.gz"
+CHECKSUM_SHA256="35ad54955a48def6a13b726da17cf9b95119b794dd9ec183a403c197d25fc4de"
+SOURCE_DIR="kdialog-v$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+commandBinDir=$binDir
+
+PROVIDES="
+	kdialog$secondaryArchSuffix = $portVersion
+	app:kdialog$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	cmd:dbus_launch
+	lib:libKF5Auth$secondaryArchSuffix
+	lib:libKF5Bookmarks$secondaryArchSuffix
+	lib:libKF5DBusAddons$secondaryArchSuffix
+	lib:libKF5GuiAddons$secondaryArchSuffix
+	lib:libKF5Codecs$secondaryArchSuffix
+	lib:libKF5Completion$secondaryArchSuffix
+	lib:libKF5I18n$secondaryArchSuffix
+	lib:libKF5IconThemes$secondaryArchSuffix
+	lib:libKF5ConfigCore$secondaryArchSuffix
+	lib:libKF5ConfigWidgets$secondaryArchSuffix
+	lib:libKF5Notifications$secondaryArchSuffix
+	lib:libKF5Solid$secondaryArchSuffix
+	lib:libKF5SonnetCore$secondaryArchSuffix
+	lib:libKF5TextWidgets$secondaryArchSuffix
+	lib:libKF5CoreAddons$secondaryArchSuffix
+	lib:libKF5ItemViews$secondaryArchSuffix
+	lib:libKF5JobWidgets$secondaryArchSuffix
+	lib:libKF5KIOCore$secondaryArchSuffix
+	lib:libKF5Service$secondaryArchSuffix
+	lib:libKF5WidgetsAddons$secondaryArchSuffix
+	lib:libKF5WindowSystem$secondaryArchSuffix
+	lib:libKF5XmlGui$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5PrintSupport$secondaryArchSuffix
+	lib:libQt5Widgets$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
```

Comment:
Are all of the Qt/KDE requirements needed? On my Linux box `kdialog` isn't linked against many of these.

Reply:
Those are the requirements that it needed when compiling it, I added them one by one as compilation failed and I retried with the added dependency